lib/owdit.js

```diff
@@ -2,9 +2,46 @@ const exec = require("executive");
 const formatter = require("./formatters/default")
 const Client = require("node-rest-client").Client;
 const util = require("util");
+const fs = require("fs");
+const path = require("path");
+const rcFilename = ".owditrc";
+
+// Read from .owditrc the lists of packages that should be excluded or only generate warnings
+function readWarnsAndExcludes(cwd) {
+  let warns = [];
+  let excludes = [];
+  const rcFile = path.resolve(cwd, rcFilename);
```

Comment:
So my assumption was wrong: it's technically perfectly possible to `require()` a JSON file, but the file must have a `.json` extension.
I tried to require `.owditrc` and this resulted in an error as node assumed Javascript. `require.extensions` could allow `.owditrc` to be considered as JSON but is now deprecated.
And of course "owdit.json" is not as sexy.